src/laboratory-link.component.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { ConfigurableLink } from "@openmrs/esm-framework";
+import { useTranslation } from "react-i18next";
+
+export default function LaboratoryLink() {
+  const { t } = useTranslation();
+  return (
+    <ConfigurableLink to={`${window.spaBase}/home/laboratory`}>
+      {t("laboratory", "Laboratory")}
+    </ConfigurableLink>
+  );
```

Comment:
Not sure this ticket was specified properly in regards to interactions because it conflicts with the current design paradigms 

Based on the way the other apps in the app menu work, if it's moved to the app menu then we should remove it from the home sidenav and have it launch the same way `dispensing` and `fast data entry` launch

Reply:
Agree @pirupius. How about having under the system admin like cohort builder?